src/python/pants/option/options_bootstrapper_test.py

```diff
@@ -407,70 +407,6 @@ def test_setting_pants_config_in_config(self, tmp_path: Path) -> None:
         logdir = ob.get_bootstrap_options().for_global_scope().logdir
         assert "logdir1" == logdir
 
-    def test_alias(self, tmp_path: Path) -> None:
```

Comment:
This is now a test on Options, instead of on OptionsBootstrapper, so it has moved (suitably modified) to options_test.py